src/PackageJsonSynchronizer.php

```
@@ -149,12 +149,23 @@ private function resolveImportMapPackages($phpPackage): array
         $dependencies = [];
 
         foreach ($packageJson->read()['symfony']['importmap'] ?? [] as $importMapName => $constraintConfig) {
-            if (\is_array($constraintConfig)) {
-                $constraint = $constraintConfig['version'] ?? [];
-                $package = $constraintConfig['package'] ?? $importMapName;
-            } else {
+            if (\is_string($constraintConfig)) {
+                // Case "jquery": "^3.0" | "@mybundle/script.js": "path:%PACKAGE%/script.js"
                 $constraint = $constraintConfig;
                 $package = $importMapName;
+                $entrypoint = false;
+            } else {
```

Let's check if `$constraintConfig` is an array before accessing properties from it, like it was done before.

Then, we can add a final `else` that throws about `$constraintConfig` invalid type

---

Should we throw an exception or bypass the current package sync, maybe with a warn ?

I guess it's better to not stop all the flex process when a bundle didn't do properly the configuration. WDYT ?

---

If a package contains invalid entries, I think it's better to explicitly throw an exception.